### ISSUE TITLE
[FEATURE REQUEST] Accessibility status/instructions in Wiki

### ISSUE THREAD

**Issue by bitbinge** [Ultramarine-Linux/ultramarine#38]
**Is your feature request related to a problem? Please describe.**
Yes.  Fedora (& by proxy Ultramarine) does not have a good guide for accessibility, especially workarounds for Wayland.

**Describe the solution you'd like**
If inline with the goals of Ultramarine, I would like to contribute to this documentation, and ideally, for common concerns, work with Ultramarine developers to incorporate sane defaults for accessibility.

**Describe alternatives you've considered**
N/A.

**Additional context**
Good examples of accessibility guides are the [Accessibility section in the Ubuntu Desktop Guide](https://help.ubuntu.com/stable/ubuntu-help/a11y.html.en) and the [Accessibility page in the Debian wiki](https://wiki.debian.org/accessibility).
